docs/source/api/v5/cdns.rst

```diff
@@ -89,15 +89,15 @@ Response Structure
 			"dnssecEnabled": false,
 			"domainName": "-",
 			"id": 1,
-			"lastUpdated": "2018-11-14 18:21:06+00",
+			"lastUpdated": "2018-11-14T18:21.516553Z",
```

Comment:
Is `516553` seconds? Shouldn't it be two digits and have a `:` after minute? or are we missing seconds and this is milliseconds?

Reply:
You are right, it is missing the seconds